src/pages/create-course/CreateCourse.tsx

```diff
@@ -42,7 +80,7 @@ const CreateCourse = () => {
 
   const createNewSection = () => {
     const newSectionData = { ...sectionInitialData }
-    newSectionData.id = data.sections.length
+    newSectionData.id = Date.now() - data.sections.length
```

Comment:
Why do we need to assign an id?

Reply:
To know which sections were changed and for drop and drag.

Reply:
I suppose `newSectionData.id = Date.now()` will be sufficient for `newSectionData.id` or use data of sections length ? 
why are you subtracting the `data.sections.length` for id ?
```suggestion
    newSectionData.id = Date.now()
```

Reply:
This line was created from this PR https://github.com/ita-social-projects/SpaceToStudy-Client/pull/1455 to not wait for merging. You can discuss this there. But I think you're right, maybe we can you without data.sections.length.